doc/xds-test-descriptions.md

```diff
@@ -745,3 +745,36 @@ the forwarding rule port `80` and the URL map host rule `myservice`.
 1. No traffic goes to backends when configuring the target URI
 `xds:///myservice`, the forwarding rule port `80` and the host rule 
 `myservice::80`.
+
+### outlier_detection
+This test verifies that the client applies the outlier detection configuration
+and temporarily drops traffic to a server that fails requests.
+
+Client parameters:
+
+1.  --num_channels=1
+2.  --qps=100
```

Comment:
That is not reflected in any other test case in this file, so I don't think it makes sense to change it just for this test.

Reply:
Yea, this document is out of date. In the PR description, you mentioned that "We need to have enough QPS so that within the configured interval" - so in your case 100 might be reasonable. Let's keep it as is, and, after it works, we can check if 25 qps works and is as stable as 100.